### DISCUSSION TITLE
Gramps Web try access private S3 storage from FE

### DISCUSSION THREAD

**Discussion by Illia-M** [gramps-project/gramps-web#530] [Announcements]
On map view I see this strange request, I think that storage of media should be hidden behind the BE part(Gramps Web API) or not?
![image](https://github.com/user-attachments/assets/1feaaff8-a808-4319-86d3-2bcd935f6c8b)
![Знімок екрана 2024-11-20 о 13 49 18](https://github.com/user-attachments/assets/b726636b-4e56-4166-8f0b-af017d6dcf21)

Debian
Gramps 5.2.3
Gramps Web API 2.5.2
Gramps Web Frontend 24.10.0
Gramps QL 0.3.0
Sifts 1.0.0
locale: uk
multi-tree: false
task queue: true
OCR: true
chat: true

P.S. Sorry for Ukrainian

**Comment by DavidMStraub** [comment]
When storing media files on S3, they are directly sent to the client using [pre-signed URLs](https://docs.aws.amazon.com/AmazonS3/latest/userguide/ShareObjectPreSignedURL.html).

**Comment by DavidMStraub** [comment]
Btw, please use the forum or discussion page if you have questions.
